backend/apps/owasp/api/search/committee.py

```
@@ -6,7 +6,17 @@
 
 def get_committees(query, attributes=None, limit=25, page=1):
-    """Return committees relevant to a search query."""
+    """Return committees relevant to a search query.
+
+    Args:
+        query (str): The search query string.
+        attributes (list, optional): List of attributes to retrieve. Defaults to None.
+        limit (int, optional): Number of results per page. Defaults to 25.
+        page (int, optional): Page number for pagination. Defaults to 1.
+
+    Returns:
```

Ruff (0.8.2)

Missing blank line after last section ("Returns")

Add blank line after "Returns"

(D413)